products/chef-infra-server.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Chef Infra Server
+category: server-app
+tags: erlang-runtime ruby-runtime
+iconSlug: chef
+permalink: /chef-infra-server
+alternate_urls:
+-   /chef-server
+versionCommand: chef-server-ctl version
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: https://docs.chef.io/release_notes_server/#__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   repology: chef-infra-server
+
+auto:
+  methods:
+  -   custom: chef-infra-server
+
+# support(x) = releaseDate(x+1)
+# eol(x) = releaseDate(x+2) or the date documented on https://docs.chef.io/versions/
+releases:
+-   releaseCycle: "15"
+    releaseDate: 2022-06-13
+    support: true
+    eol: false
+    latest: "15.9.20"
+    latestReleaseDate: 2024-01-10
+
+-   releaseCycle: "14"
+    releaseDate: 2020-06-08
+    support: 2022-06-13
+    eol: false
+    latest: "14.16.19"
+    latestReleaseDate: 2022-06-07
+
+-   releaseCycle: "13"
+    releaseDate: 2019-05-31
+    support: 2020-06-08
+    eol: 2021-06-30
+    latest: "13.2.0"
+    latestReleaseDate: 2020-04-10
+
+-   releaseCycle: "12"
+    releaseDate: 2014-11-24
+    support: 2019-05-31
+    eol: 2020-12-31
+    latest: "12.19.31"
+    latestReleaseDate: 2019-03-07
+
+-   releaseCycle: "11"
+    releaseDate: 2013-09-30
+    support: 2014-11-24
+    eol: 2018-12-31
```

Comment:
Why not 2019-05-31 (the release date of 13) ?

Reply:
https://www.chef.io/blog/end-of-life-announcement-for-chef-reporting-enterprise-chef-server-11-and-chef-analytics